routes/truckRoutes.js

```diff
@@ -0,0 +1,76 @@
+// Connect to mongodb in this module as this is where you'll be making
+//creat/read/delete calls to your database
+// use 'mongodb://localhost/foodTruckAPI' for your mongoose connection string
+var express = require('express');
+var mongoose = require('mongoose');
+var foodKind;
+/*eslint-disable */
+var db = mongoose.connect('mongodb://localhost/foodTruckAPI');
+/*eslint-enable */
+var app = express();
```

Comment:
This isn't necessary. You already have app instantiated in index.js. If you want to process params, you can do it directly on the router instance:

`router.param('type', function (request, response, next) {`